generators/app/index.js

```diff
@@ -28,17 +42,37 @@ module.exports = class extends Generator {
   }
 
   writing() {
+    this.log(`project: ${this.props.project}`);
+    this.log(`organization: ${this.props.organization}`);
+    this.log(`git_domain: ${this.props.git_domain}`);
 
-    this.fs.copyTpl(glob.sync(this.templatePath('**/*'), { dot: true }), this.destinationPath(), {/*context*/});
-
-    // this.fs.copyTpl(
-    //   this.templatePath('index.html'),
-    //   this.destinationPath('public/index.html'),
-    //   { title: 'Templating with Yeoman' }
-    // );
+    this.fs.copyTpl(
+      glob.sync(this.templatePath("**/*"), { dot: true }),
+      this.destinationPath(),
+      this.props
+    );
   }
 
-  install() {
-    this.installDependencies();
+  end() {
+    this.spawnCommandSync("git", ["init", "--quiet"], {
+      cwd: this.destinationPath()
+    });
+
+    this.spawnCommandSync("git", ["add", "."], {
```

Comment:
I suggest make this a function and we make git optional
also better we make sure it builds before commiting ?

Reply:
Added --nogit argument
Then I think we can keep it simple, if the result is not perfect the operator will fix and eventually PR to the generator.